registry/time/functions.go

```diff
@@ -44,28 +49,32 @@ func (tr *TimeRegistry) Date(fmt string, date any) (string, error) {
 //
 // TODO: Change signature
 func (tr *TimeRegistry) DateInZone(fmt string, date any, zone string) (string, error) {
-	var t time.Time
+	t := computeTimeFromFormat(date)
+	loc, err := time.LoadLocation(zone)
+	if err != nil {
+		return t.In(time.UTC).Format(fmt), err
+	}
+
+	return t.In(loc).Format(fmt), nil
+}
+
+// computeTimeFromFormat returns a time.Time object from the given date.
+func computeTimeFromFormat(date any) time.Time {
```

Comment:
This function should be placer in `helpers.go` and have her tests in `helpers_test.go`